### PR TITLE
chore(js): deny lint warnings in CI

### PR DESCRIPTION
**Pull request by hyf0**
### Test Plan

---

### CODE REVIEW

**Comment by hyf0** [comment]
Current dependencies on/for this PR:
* main
  * **PR #315** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/315?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #316** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/316?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #317** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/317?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #319** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/319?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #320** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/320?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #321** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/321?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
              * **PR #322** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/322?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                * **PR #323** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/323?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                  * **PR #324** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/324?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                    * **PR #325** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/325?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                      * **PR #326** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/326?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                        * **PR #327** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/327?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                          * **PR #328** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/328?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                            * **PR #329** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/329?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
                              * **PR #332** <a href="https://app.graphite.dev/github/pr/rolldown-rs/rolldown/332?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/321?utm_source=stack-comment).

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -151,7 +151,11 @@ jobs:
         run: yarn lint-filename
 
       - name: Oxc Lint
-        run: yarn lint
+        # Run `yarn lint` to get display for warnings 
+        # Run `yarn lint:ci` to get exit code 1 for warnings
```
Comment (hyf0) [inline .github/workflows/ci.yaml:155]: @Boshen cc. You might want to improve this usage. Deny and display warnings in one go.

Comment (Boshen) [reply .github/workflows/ci.yaml:155]: Let me take a look

Comment (Boshen) [reply .github/workflows/ci.yaml:155]: https://github.com/oxc-project/oxc/issues/1403

File: packages/node/test/runner.ts [hunk 2]
```diff
@@ -24,12 +24,8 @@ function runCases() {
...
 
         test(subCaseName, async () => {
-          try {
-            await runCaseBundle(caseRoot, config)
-            expect(true)
-          } catch (error) {
-            throw error
-          }
+          await runCaseBundle(caseRoot, config)
+          expect(true)
```
Comment (underfin) [inline packages/node/test/runner.ts:28]: I ready changed this code, please see https://github.com/rolldown-rs/rolldown/pull/314

Comment (hyf0) [reply packages/node/test/runner.ts:28]: Great. You could change it back in https://github.com/rolldown-rs/rolldown/pull/314.

--- edited

@underfin The code is related to the js lint ci status, could we merge this first and then I will solve the conflict?

Comment (underfin) [reply packages/node/test/runner.ts:28]: ok

**Review by underfin — APPROVED** [review]

**Comment by hyf0** [comment]
### Merge activity

* **Nov 19, 6:40 AM**: @hyf0 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/321).
* **Nov 19, 6:42 AM**: [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/321) rebased this pull request as part of a merge.
* **Nov 19, 6:43 AM**: @hyf0 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/rolldown-rs/rolldown/321).

**Pull request merged**